EIPS/eip-831.md

```diff
@@ -0,0 +1,42 @@
+## Preamble
+
+    EIP: 831
+    Title: URL Format for Ethereum
+    Author: ligi <ligi@ligi.de>
+    Type: Standard Track
+    Category: ERC
+    Status: Draft
+    Replaces: 67
```

Comment:
@nicksavers is this how to refer to the original GitHub issue?  Or should the issue be mentioned in Motivation?

Reply:
context: https://github.com/ethereum/EIPs/blame/master/EIPS/eip-1.md#L134

Reply:
Does this refer to issue https://github.com/ethereum/EIPs/issues/67? 
It doesn't really replace a previously adopted EIP, so I don't see the need for adding it to the EIP. Besides, that issue is still open. If this PR is supposed to supersede it, shouldn't it be closed?

Reply:
was inspired by how it was done in 681 - will remove it then